pydis_core/utils/interactions.py

```diff
@@ -23,9 +23,10 @@ def user_has_access(
         allowed_users: A sequence of user ids that are allowed access
         allowed_roles: A sequence of role ids that are allowed access
     """
-    if user.id in allowed_users or any(role.id in allowed_roles for role in getattr(user, "roles", [])):
-        return True
-    return False
+    return bool(
```

Comment:
why `bool(`?

Reply:
It's just what ruff auto-corrected it to. Just a bracket would probably work too, but I found this more readable.